app/controllers/concerns/permitted_development_rights.rb

```diff
@@ -12,6 +12,6 @@ def set_permitted_development_rights
   end
 
   def planning_applications_scope
-    current_local_authority.planning_applications.includes(:permitted_development_rights)
```

Comment:
This not needed?

Thought it would help loading all the previous responses because we need to iterate through this collection